preload_data.sh

```diff
@@ -0,0 +1,1735 @@
+#!/usr/bin/env bash
+#
+# Copyright 2023 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# set -o errexit
+set -o pipefail
+set -o errtrace
+# set -o nounset
+
+OC=oc
+YQ=yq
+FROM_NAMESPACE=$1
+TO_NAMESPACE=$2
```

Comment:
```suggestion
FROM_NAMESPACE=""
TO_NAMESPACE=""
```

Cannot just set it to positional args $1 and $2 because there are other optional flags, so if executed like this `./preload_data.sh --oc oc/path ns1 ns2`, then FROM_NAMESPACE will actually equal `--oc` and TO_NAMESPACE will actually equal `oc/path`, which is not correct